src/common/pf_lldp.c

```diff
@@ -466,6 +466,28 @@ static void pf_lldp_add_management (
    pf_put_byte (0, PF_FRAME_BUFFER_SIZE, p_buf, p_pos);
 }
 
+/**
+ * Mark lldp peer data as invalid.
+ *
+ * @param net              InOut: The p-net stack instance
+ * @param loc_port_num     In:    Local port number.
+ *                                Valid range: 1 .. PNET_MAX_PORT
+ */
+void pf_lldp_peer_timeout(pnet_t * net, int loc_port_num)
+{
+   pf_port_t * p_port_data = pf_port_get_state (net, loc_port_num);
+
+   os_mutex_lock (net->lldp_mutex);
+   p_port_data->lldp.is_peer_info_received = false;
+   p_port_data->lldp.peer_info.chassis_id.is_valid = false;
+   p_port_data->lldp.peer_info.port_id.is_valid = false;
+   p_port_data->lldp.peer_info.port_description.is_valid = false;
+   p_port_data->lldp.peer_info.management_address.is_valid = false;
+   p_port_data->lldp.peer_info.port_delay.is_valid = false;
+   p_port_data->lldp.peer_info.phy_config.is_valid = false;
+   os_mutex_unlock (net->lldp_mutex);
```

Comment:
Set `ttl `to zero, as that seems to mean that it is invalid.

Reply:
Done